test/unit/yaml.spec.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Tests for yaml module.
+ * @file yaml.spec.js
+ */
+
+'use strict';
+
+// Setup chai.
+const chai = require('chai');
+const expect = chai.expect;
+const filesystem = require('mock-fs');
+const fs = require('fs-extra');
+const hasher = require('object-hash');
+const os = require('os');
+const path = require('path');
+chai.should();
+
+// Get yaml module to test
+const yaml = require('./../../lib/yaml')({error: () => {}});
+
+// This is the file we are testing
+describe('yaml', () => {
```

Comment:
same issue, I think is one area where we have to avoid arrow functions because of lexical scope binding.